content/10.diagnostics.md

```diff
@@ -331,14 +339,12 @@ The rapid development of molecular tests has provided a valuable, albeit imperfe
 
 Although several molecular diagnostic tests to detect viral genetic material have high selectivity and sensitivity, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
 Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
-In such contexts, serological tests are more informative.
+In such contexts, serological tests are informative.
 Serological tests use many of the same technologies as the immunoassays used to detect the presence of an antigen, but are instead used to evaluate the presence of antibodies against SARS-CoV-2 in a serum sample.
-These tests that assay antibodies are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
-
+These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
 Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
-In the process of mounting a response to a pathogen, the adaptive immune system produces antibodies specific to the pathogen.
 Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
-The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
+The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.<!--Needs Citation-->
```

Comment:
Book is also added as a manual ref under URL, so see if this works. If it does, can remove.

```suggestion
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @isbn:9780815336426].
```